store/localstore/kv.go

```diff
@@ -139,6 +143,7 @@ func (s *dbStore) Begin() (kv.Transaction, error) {
 func (s *dbStore) Close() error {
 	mc.mu.Lock()
 	defer mc.mu.Unlock()
+	s.gc.Stop()
```

Comment:
Call gc.Stop(), delete worker maybe not exist before u call s.db.Close(), then gc.store.db.Commit seems not work well.